survey_form/index.html

```diff
@@ -43,8 +43,94 @@
 
     <div class="tittle"><h1 class="tittle__item">Survey Form</h1></div>
 
-    <form class="form">
-      <p class="form__tittle">Lets us now how we can improve freeCodeCamp</p>
+    <form class="form-survey">
+      <legend class="form-survey__title">
+        Lets us now how we can improve freeCodeCamp
+      </legend>
+
+      <label class="form-survey__label" for="name">Name: </label>
+      <input
+        type="text"
+        id="name"
+        class="form-survey__input"
+        required
+        placeholder="Leonardo DiCaprio"
+      />
+
+      <label class="form-survey__label" for="email">Email: </label>
+      <input
+        type="email"
+        id="email"
+        class="form-survey__input"
+        required
+        placeholder="leonardo.dicaprio@gmail.com"
+      />
+
+      <label class="form-survey__label" for="age">Age: </label>
+      <input
+        type="age"
+        id="number"
+        class="form-survey__input"
+        placeholder="44"
+        min="12"
+        max="123"
+        required
+      />
+
+      <label class="form-survey__label" for="current_role"
+        >Which option best describes your current role?</label
+      >
+
+      <select class="form-survey__dropdown" id="current_role">
+        <option value="select_an_option" disabled>Select an option</option>
+        <option value="student">Student</option>
+        <option value="full_time_job">Full time job</option>
+        <option value="full_time_learner">Full time learner</option>
+        <option value="prefer_not_to_say">Prefer not to say</option>
+        <option value="other">Other</option>
+      </select>
+
+      <label
+        class="form-survey__label form-survey__label--radiobutton"
+        for="form-survey__radiobutton"
+        >How likely is that you would recommend freeCodeCamp to a friend?</label
+      >
+
+      <div class="form-survey__radiobutton" id="form-survey__radiobutton">
+        <div class="form-survey__radiobutton--group">
+          <input
+            class="form-survet__input--radiobutton"
+            type="radio"
+            name="definitely"
+            id="definitely"
+          />
+          <label class="form-survey__input--label" for="definitely"
```

Comment:
Ta klasa też potrzebuje korekty, bo jest przeznaczona dla `inputa`, a użyta w labelu, w innych miejscach to samo.

Reply:
Nie rozumiem, mógłbyś rozwinąć ?

Reply:
Chodzi mi o to, że w labelu użyłeś klasy `form-survey__input--label`, ta klasa jest przeznaczona dla inputa, nie labela. Do tego modyfikator `--label` nie ma tutaj zbytnio sensu, bo to tak jakbyś dodawał style, które modyfikują input, by stał się labelem. Także lepiej użyć klasy `form-survey__label` (ewentualnie dodać do tego jakiś modyfikator, jeżeli jest potrzeba). Mam nadzieję, że teraz jaśniej :)

Reply:
Yup, wszystko jasne :)